### ISSUE TITLE
Comment after a selector and a comma causes an error

### ISSUE THREAD

**Issue by mzgol** [sass/libsass#1022]
The following SCSS:
```scss
button, /* 1 */
div {
    top: 10px;
}
```
causes an error:
```
>> invalid top-level expression
>>   Line 1  Column 1  app/core-styles/_normalize.scss
```
node-sass `3.0.0-beta.4`. Ruby Sass parses it fine.

Related: #890, #1021.

**Comment by mgreter** [comment]
Already fixed in latest master and 3.2.0-beta.4 (node-sass 3.0.0-beta.4 uses libsass 3.2.0-beta.2) !

**Comment by mzgol** [comment]
Great, thanks! :)
